app/src/docs/v1.api-spec.yaml

```diff
@@ -1049,6 +1375,47 @@ components:
         description: a version identifier created in S3
         example: '1647462569641'
   schemas:
+    DB-Bucket:
+      allOf:
+        - type: object
+          required:
+            - accessKeyId
+            - active
+            - bucket
+            - bucketId
+            - bucketName
+            - endpoint
+            - key
+            - region
+            - secretAccessKey
+          properties:
+            accessKeyId:
+              example: REDACTED
+            active:
+              type: boolean
+              description: Determines whether this bucket is considered active
+              default: true
+              example: true
+            bucket:
+              type: string
+            bucketId:
+              type: string
+              description: The primary identifier for this bucket
+              format: uuid
+              example: ac246e31-c807-496c-bc93-cd8bc2f1b2b4
+            bucketName:
+              type: string
+            endpoint:
+              type: string
+            key:
+              type: string
+              description: The prefix to path of all objects in the bucket
+              example: coms/env
+            region:
+              type: string
+            secretAccessKey:
+              example: REDACTED
```

Comment:
When defining properties, make sure to always include a description and an example wherever possible. While this is "sufficient" for OpenAPI spec definition, we generally like to go the extra mile and make sure everything is relentlessly documented especially for component definitions.